### ISSUE TITLE
Relation of input and output files 

### ISSUE THREAD

**Issue by bernt-matthias** [OpenMS/OpenMS#4402]
There are some tools that that have `<ITEMLIST>` in and outputs that are apparently related to each other: 

- SpectraSTSearchAdapter -spectra_files -output_files
- SpecLibSearcher -in -out
- SeedListGenerator -in -out
- MapAlignerSpectrum -in -out (or -trafo_out)
- MapAlignerPoseClustering  -in -out (or -trafo_out)
- MapAlignerIdentification -in -out (or -trafo_out)
- FileMerger -in -trafo_out

I would like to know if the inputs are treated independently of each other for these tools. That is if 

`TOOL -in 1.in 2.in 3.in -out 1.out 2.out 3.out` is equivalent to 

```
TOOL -in 1.in -out 1.out
TOOL -in 2.in -out 2.out
TOOL -in 3.in -out 3.out
```

That is three separate calls to the respective tool.

**Comment by bernt-matthias** [comment]
According to https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/html/TOPP_example_mapalignment.html `MapAligner* ` are not independent.

FileMerger as well not independent

**Comment by jpfeuffer** [comment]
Yes. A quick check and my background knowledge confirm your results.
FileMerger and MapAligners consider the whole data.
For the others I think the main reason is not to load common things like the spectrum database.
